server/command_hooks.go

```diff
@@ -183,6 +195,17 @@ func (p *Plugin) executeCommandEphemeral(args *model.CommandArgs) *model.Command
 	return &model.CommandResponse{}
 }
 
+func (p *Plugin) executeCommandEphemeralOverride(args *model.CommandArgs) *model.CommandResponse {
+	_ = p.API.SendEphemeralPost(args.UserId, &model.Post{
+		ChannelId: args.ChannelId,
+		Message:   "test ephemeral override",
+		Props: model.StringInterface{
+			"type": "custom_demo_plugin",
```

Comment:
I had the same confusion as @hanzei of what I should be seeing when I ran the command.

It would be best if we can add another type of post that is different from the one used in the regular posts so the example can be more obvious that the ephemeral messages can have their own custom types as well.

Reply:
@marianunez  I updated the example to use it's own post_type aside from the existing post_type, though borrowed most of the content.  Does the following look ok?
<img width="557" alt="Screen Shot 2019-06-25 at 8 31 40 PM" src="https://user-images.githubusercontent.com/2914443/60149261-ef70bc00-9788-11e9-9b44-e5f972e61be5.png">

Reply:
This looks great @scottleedavis!